docs/01-Using-Fleet/03-REST-API.md

```diff
@@ -3478,16 +3488,61 @@ For example, a policy might ask “Is Gatekeeper enabled on macOS devices?“ Th
 
 ### Add policy
 
+There are two ways of adding a policy:
+1. by setting "name", "query", "description". This is the preferred way.
+2. (Legacy) re-using the data of an existing query, by setting "query_id". If "query_id" is set,
+then "query" must not be set, and "name" and "description" are ignored.
+
+An error is returned if both "query" and "query_id" are set on the request.
+
 `POST /api/v1/fleet/global/policies`
 
 #### Parameters
 
-| Name       | Type    | In   | Description                           |
-| ---------- | ------- | ---- | ------------------------------------- |
-| query_id   | integer | body | **Required.** The query's ID.         |
-| resolution | string  | body | The resolution steps for the policy.  |
+| Name        | Type    | In   | Description                          |
+| ----------  | ------- | ---- | ------------------------------------ |
+| name        | string  | body | The query's name.                    |
```

Comment:
```suggestion
| name        | string  | body | **Required.** The query's name.                    |
```

Reply:
Actually not required if the user uses the `query_id` argument, see above:
```
There are two ways of adding a policy:
1. by setting "name", "query", "description". This is the preferred way.
2. (Legacy) re-using the data of an existing query, by setting "query_id". If "query_id" is set,
then "query" must not be set, and "name" and "description" are ignored.

An error is returned if both "query" and "query_id" are set on the request.
```